compatibility/BUILD

```diff
@@ -69,6 +74,25 @@ config_setting(
        platform_version == "0.0.0"
 ]
 
+[
+    daml_trigger_dar(sdk_version)
+    for sdk_version in sdk_versions
+]
+
+[
+    daml_trigger_test(
+        compiler_version = sdk_version,
+        runner_version = platform_version,
+    )
+    for sdk_version in sdk_versions
+    for platform_version in platform_versions
+    # Test that the DAML trigger runner can run DARs built with an older SDK
+    # version. I.e. where the runner version is at least the SDK version or
+    # more recent. The HEAD version 0.0.0 is a special case.
+    if versions.is_at_least(sdk_version, platform_version) and sdk_version != "0.0.0" or
```

Comment:
Can we use the version comparison I’ve written here?

Reply:
Done, I've noticed that their arguments were in backwards order compared to the [skylib ones](https://github.com/bazelbuild/bazel-skylib/blob/4b25373d12887f5add565197c4a163e9f1d9b716/lib/versions.bzl#L57-L67), I've aligned them for consistency.